include/mesh/mesh_function.h

```diff
@@ -105,11 +105,15 @@ class MeshFunction : public FunctionBase<Number>,
    * own and specifying the Trees::NODES method.  specifies the method
    * to use when building a \p PointLocator
    */
-  virtual void init () override { this->init(Trees::NODES); }
+  virtual void init () override;
```

Comment:
The comment here is atavistic now, isn't it?

Reply:
Yes, I'll fix that.